core/src/martian/schema.cljc

```diff
@@ -85,21 +85,26 @@
               {(cond-> (keyword name)
                  (not required)
                  s/optional-key)
-               (make-schema definitions param)}))
+               (make-schema definitions {} param)}))
        (into {})))
 
-(defn- resolve-ref [definitions ref]
-  (some->> ref
-           (re-find #"#/definitions/(.*)")
-           second
-           keyword
-           definitions))
+(defn- resolve-ref [definitions parameters ref]
```

Comment:
It feels like 'definitions' and 'parameters' are two categories of the same thing. I'd be tempted to merge them, either directly (risk of key collision?) or to provide it as a map that looks like `{:definitions {...} :parameters {...}}` and call it `ref-lookup`

Reply:
Please review the most recent comment.  Thank you.